.buildkite/pipeline.yml

```
@@ -5,4 +5,4 @@ agents:
 
 steps:
   - label: ":wave: Greetings!"
-    command: "echo 'My first EUI pipeline!'"
+    command: "echo 'The official EUI pipeline.'"
```

[incredibly optional comment, this is totally me bikeshedding] 'Official' sounds a bit odd here for some reason (it implies the existence of a _non_-official pipeline out there, like a twitter impersonator 🤣)

How about just something very boring like
```suggestion
    command: "echo 'The EUI Buildkite pipeline'"
```
```suggestion
    command: "echo 'The EUI CI pipeline'"
```
or if you want to get snarky, something like

```suggestion
    command: "echo 'The EUI CI pipeline, now with more robots'"
```